src/controllers/midi/midicontroller.h

```diff
@@ -57,12 +57,24 @@ class MidiController : public Controller {
                          unsigned char value);
 
   protected:
-    Q_INVOKABLE void sendShortMsg(unsigned char status, unsigned char byte1, unsigned char byte2);
+    Q_INVOKABLE virtual void sendShortMsg(unsigned char status,
+                                          unsigned char byte1, unsigned char byte2) = 0;
     // Alias for send()
-    Q_INVOKABLE inline void sendSysexMsg(QList<int> data, unsigned int length) {
-        send(data, length);
+    Q_INVOKABLE inline void sendSysexMsg(QList<int> data, unsigned int length = 0) {
+        Q_UNUSED(length);
```

Comment:
The same here, all the length and int to char looks scary. 
I think we should either keep the  Q_INVOKABLE interface untouched or clean it up entirely. 
Why using a QList<int> if we actually send a byte array?